mewarpx/tests/test_custom_criteria.py

```diff
@@ -11,7 +11,7 @@ def check_particle_nums():
 def add_particles():
     from pywarpx import _libwarpx
     # add 10,000 particles
-    _libwarpx.add_particles(x=np.zeros((10000)), y=np.zeros((10000)), z=np.zeros((10000)))
+    _libwarpx.add_particles(species_name="electrons", x=np.zeros((10000)), y=np.zeros((10000)), z=np.zeros((10000)))
```

Comment:
`zeros` can accept an int or a tuple for a shape, but if you wanted to pass a tuple it needs to be `(10000, )`; the parentheses alone weren't doing anything.
```suggestion
    _libwarpx.add_particles(species_name="electrons", x=np.zeros(10000), y=np.zeros(10000), z=np.zeros(10000))
```